.pre-commit-config.yaml

```diff
@@ -46,11 +46,6 @@ repos:
         args: ["--fix", "--show-fixes"]
       - id: ruff-format
 
-  - repo: https://github.com/codespell-project/codespell
```

Comment:
codespell is usually a "low maintenance" dependency, was the amount of false positive too large ?

Reply:
I think I just didn't want to do a bunch of spelling corrections -- the main goal was to get the package published. But I guess if it makes future maintenance easier I can bring back codespell and do the corrections